Implemented KAN-36

SurajCheema opened the pull request:
Delete all passwords securely.

SurajCheema committed:
Implemented KAN-36

Delete all passwords securely.

Takenobou reviewed (changes requested):
dont edit any files related to other user stories. extend them in your own class. so dont edit Credential.java (you just added comments for irrelevent code?) and make a new class called DeleteStorage that does what you want.

add your ide config to the gitignore, that shouldnt be commited

SurajCheema committed:
KAN-36 Corrections

Delete all credentials. Corrected layout, following file pattern. Added to gitignore.
